pom.xml

```diff
@@ -83,7 +83,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <dropwizard.version>0.7.1</dropwizard.version>
-        <protobuf.version>2.6.1</protobuf.version>
+        <protobuf.version>2.4.1</protobuf.version>
```

Comment:
Sure, the changes that come along within this pull request allow easy transition to a different version of protobuf, without the need of any code change. I just thought that there might be other people who want to use it with protobuf 2.4.1, otherwise I can always release and put it in our internal repo.